website/docs/components/table/partials/guidelines/guidelines.md

```diff
@@ -155,4 +155,54 @@ Row placement determines the visual styling based on where the row is placed rel
 - We recommend using `medium` cell density by default.
 - If content is complex or a smaller data set (e.g., a Table of basic user data), `tall` cell density allows for more breathing room surrounding the content.
 - If content is largely string/text-based, `short` allows for more content to be displayed within the page.
-- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
\ No newline at end of file
+- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
+
+## Multi-Selection
+
+Multi-selection enables users to select multiple rows in a table, facilitating bulk operations, actions, and editing.
+
+Multi-selection consist of: 
+
+1. `Header Column / Selection` Component:
+Used in the header row of the Table. This acts as a master control, enabling end-users to either select or deselect all rows simultaneously.
+
+
+![Example of centered content within a table](/assets/components/table/multi-select-header.png)
+
+2. `Cell / Selection` Component:
+Used in the start position each Table row. It allows for individual row selection and aligns with the Header Column component, reflecting the overall selection status of the table.
+
+![Example of centered content within a table](/assets/components/table/multi-select-cells.png)
+
+!!! Info 
+
+In the Ember component, unlike the `cell/select` Figma component, achieving an indeterminate state is not possible. 
+!!!
+
+### Intended interaction
+
+#### Within a Simple Table (Single Page):
+
+- Selecting an Individual Row: Indicates that the row is ready for actions such as editing or deletion.
+- Interaction with the Top-Level Header Checkbox: Non-Checked State: Initially, the checkbox is unmarked when none of the rows are selected.
+- Checkbox Activation: Clicking the checkbox selects every row in the table, including any previously chosen individually, ensuring collective selection.
+- Indeterminate State: The checkbox shows an indeterminate state when some but not all rows are selected, indicating a partial selection.
+
+<video width="100%" controls loop>
+  <source
+    src="/assets/components/table/multi-selection-interaction-01.mp4"
+    type="video/mp4"
+  />
+</video>
+
+#### Within a paginated table: 
+
+- Global Actions:
+In addition to page-specific actions, paginated tables may benefit from global actions like a "Select all users" feature. This would apply to all rows across every page in the table, enabling actions on a larger scale (e.g., selecting all 53 users in a dataset or deselecting). 
+
+<video width="100%" controls loop>
+  <source
+    src="/assets/components/table/multi-selection-interaction-02.mp4"
+    type="video/mp4"
+  />
+</video>
```

Comment:
```suggestion
<!--TODO: Add in for future use of pattern documentation #### Within a paginated table: 

- Global Actions:
In addition to page-specific actions, paginated tables may benefit from global actions like a "Select all users" feature. This would apply to all rows across every page in the table, enabling actions on a larger scale (e.g., selecting all 53 users in a dataset or deselecting). 

<video width="100%" controls loop>
  <source
    src="/assets/components/table/multi-selection-interaction-02.mp4"
    type="video/mp4"
  />
</video>-->
```